nats.go

```diff
@@ -699,6 +699,19 @@ func Token(token string) Option {
 	}
 }
 
+// Nkey will set the public Nkey and the signature callback to
+// sign the server nonce.
+func Nkey(pubKey string, sigCB SignatureHandler) Option {
+	return func(o *Options) error {
+		o.Nkey = pubKey
+		o.SignatureCB = sigCB
+		if sigCB == nil {
```

Comment:
Well, technically the error says that there is a key but no callback, so you should add `pubKey != "" && ..`

Reply:
ok